pkg/controller/initializer.go

```diff
@@ -36,8 +36,12 @@ func init() {
 			} else {
 				log.Warn("OpenShift version could not be determined.")
 			}
+			if mappedVersion.Version < "4.3" {
```

Comment:
@myeung18 @tchughesiv this looks like a string compare, so I wonder what the repercussions are. For example, I imagine 4.11 < 4.3

Reply:
i just checked this with a unit test and it won't work... considers 4.3 to be greater than 4.11. That's my fault... will have to make a few changes in code besides this one.

Reply:
I am also aware the bug here. I am working on a fix. I will show probably tmr..